### PR TITLE
[1LP][RFR][NOTEST]Add manual test cases for RFE

### PR DESCRIPTION
**Pull request by valaparthvi**

### CODE REVIEW

File: cfme/tests/webui/test_general_ui.py [hunk 1]
```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
...
+    casecomponent: WebUI
+    caseimportance: medium
+    initialEstimate: 1/5h
+    setup:
+        1. Navigate to Compute > Infrastructure > Providers.
+        2. Click on Advanced Search Filter.
+    testSteps:
+        1. Create an expression with multiple types of condition. Eg: arg_1 AND arg_2 OR arg_3
+    expectedResults:
+        2. Expression must be created successfully.
```
Comment (john-dupuy) [inline cfme/tests/webui/test_general_ui.py]: Here, the expectedResults number should match the testSteps number.

Comment (valaparthvi) [reply cfme/tests/webui/test_general_ui.py]: Yes, it definitely should. :roll_eyes:

File: cfme/tests/webui/test_general_ui.py [hunk 2]
```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
...
+    caseimportance: medium
+    initialEstimate: 1/5h
+    setup:
+        1. Login to a new appliance.
+    testSteps:
+        1. Check if the welcome/landing page contains `Add a Provider` action.
+    expectedResults:
+        1. Welcome/Landing page must show the next action, ie.e `Add a Provider
+
+    Bugzilla:
```
Comment (ganeshhubale) [inline cfme/tests/webui/test_general_ui.py]: Question:
is ```ie.e```  typo error?

File: cfme/tests/webui/test_general_ui.py [hunk 3]
```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
...
+    caseimportance: medium
+    initialEstimate: 1/5h
+    setup:
+        1. Create a custom navigation menu.
+    testSteps:
+        1. Check if the custom menu is visible in the left Navigation Bar.
+    expectedResults:
+        1. Custom menu must be visible.
+
+    Bugzilla:
```
Comment (ganeshhubale) [inline cfme/tests/webui/test_general_ui.py]: @valaparthvi Please check polarion block indentation
```
def test_child_tenant_quota_enforce_via_lifecycle_cloud():
    """Test Child Quota in UI

    Polarion:
        assignee: ghubale
        casecomponent: Cloud
        caseimportance: high
        initialEstimate: 1/8h
        tags: quota
        testSteps:
            1. Create a child tenant
            2. Assign quota to child tenant
            3. Provision instance over the assigned child's quota
            4. Check whether quota is exceeded or not
    """
```

File: cfme/tests/webui/test_general_ui.py [hunk 4]
```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
...
+@pytest.mark.tier(2)
+def test_welcoming_page():
+    """
+    Polarion:
+    assignee: pvala
+    casecomponent: WebUI
+    caseimportance: medium
+    initialEstimate: 1/5h
+    setup:
+        1. Login to a new appliance.
```
Comment (ganeshhubale) [inline cfme/tests/webui/test_general_ui.py]: Check polarion block indentation.

File: cfme/tests/webui/test_general_ui.py [hunk 5]
```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
...
+        * 1660460
+    """
+    pass
+
 
-        Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1660460
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_welcoming_page():
+    """
```
Comment (ganeshhubale) [inline cfme/tests/webui/test_general_ui.py]: Add ```test_requirements``` to these test cases.

**Pull request merged**